mattermost-plugin/webapp/src/index.tsx

```diff
@@ -14,9 +14,8 @@ import {selectTeam} from 'mattermost-redux/actions/teams'
 import {SuiteWindow} from '../../../webapp/src/types/index'
 import {UserSettings} from '../../../webapp/src/userSettings'
 
-
 const windowAny = (window as SuiteWindow)
-windowAny.baseURL = '/plugins/focalboard'
+windowAny.baseURL = process.env.TARGET_IS_PRODUCT ? '/plugins/boards' : '/plugins/focalboard'
```

Comment:
I added a new environment variable for whenever we need the plugin to work differently than the product. For now, it's just used here since Boards is still mostly a plugin